bctest/input_inflow

```diff
@@ -0,0 +1,71 @@
+# This is a 2D poiseuille flow when run to steady state
```

Comment:
move these into `test/test_files` and then add to `test/CMakeLists.txt`

Reply:
yes, makes sense

Reply:
I have done this, though the test currently fails for some reason — I will be debugging that.